Auth0/SafariAuthenticationCallback.swift

```diff
@@ -22,22 +22,28 @@
 
 import UIKit
 import SafariServices
+import AuthenticationServices
 
 #if swift(>=3.2)
 @available(iOS 11.0, *)
 class SafariAuthenticationSessionCallback: AuthTransaction {
 
     var state: String?
-    var authSession: SFAuthenticationSession?
```

Comment:
By removing this, there is nothing retaining the session alive, so as soon as you accept the `Auth0.com` dialog it will fail and never launch the browser.

`Attempting to load the view of a view controller while it is deallocating is not allowed and may result in undefined behavior (<SFAuthenticationViewController: 0x7f851c82b600>)`

Reply:
AnyObject not work here? Doesn't look right pulling in ObjC